db/migrate/20220131220043_create_recipes.rb

```diff
@@ -0,0 +1,17 @@
+class CreateRecipes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :recipes do |t|
+      t.string :name, null: false
+      t.text :description
+      t.string :picture
```

Comment:
Do you need this?

Reply:
Yes, it's the migration for the recipes model. The name is a needed parameter.